__tests__/__snapshots__/feeds.ts.snap

```diff
@@ -37,9 +33,7 @@ Object {
   "blockedContentCuration": Array [],
   "blockedTags": Array [],
   "excludeSourceTypes": Array [],
-  "excludeSources": Array [
-    "fd062672-63b7-4a10-87bd-96dcd10e9613",
-  ],
```

Comment:
Just to sanity check, is this change is tests alright?

Reply:
Yap removing watercooler from being hard coded. Taken care by the feed service already